app/master/cluster_master.py

```diff
@@ -28,7 +28,7 @@ def __init__(self):
         self._logger = get_logger(__name__)
         self._master_results_path = Configuration['results_directory']
         self._all_slaves_by_url = {}
-        self._all_builds_by_id = OrderedDict()
+        self._all_builds_by_id = dict()
```

Comment:
I would just write a `{}` instead of `dict()` for consistency with the line above. :p